GW2EIEvtcParser/EIData/ProfHelpers/Revenant/RevenantHelper.cs

```diff
@@ -342,6 +342,57 @@ internal static void ComputeProfessionCombatReplayActors(AbstractPlayer player,
                     AddCircleSkillDecoration(replay, effect, color, skill, lifespan, 240, ParserIcons.EffectDropTheHammer);
                 }
             }
+
+            // Abyssal Blitz (Mines)
+            if (log.CombatData.TryGetEffectEventsBySrcWithGUID(player.AgentItem, EffectGUIDs.RevenantSpearBlitzMines2, out IReadOnlyList<EffectEvent> abyssalBlitzMines))
+            {
+                var skill = new SkillModeDescriptor(player, Spec.Revenant, BlitzMines, SkillModeCategory.ShowOnSelect);
+                foreach (EffectEvent effect in abyssalBlitzMines)
+                {
+                    (long, long) lifespan = effect.ComputeDynamicLifespan(log, 7000);
+                    AddCircleSkillDecoration(replay, effect, color, skill, lifespan, 120, ParserIcons.EffectAbyssalBlitzMines);
+                }
+            }
+
+            // Abyssal Blot
+            if (log.CombatData.TryGetEffectEventsBySrcWithGUID(player.AgentItem, EffectGUIDs.RevenantSpearAbyssalBlot, out IReadOnlyList<EffectEvent> abyssalBlots))
+            {
+                var skill = new SkillModeDescriptor(player, Spec.Revenant, AbyssalBlot, SkillModeCategory.CC);
+                foreach (EffectEvent effect in abyssalBlots)
+                {
+                    (long, long) lifespan = effect.ComputeLifespan(log, 3000);
+                    AddCircleSkillDecoration(replay, effect, color, skill, lifespan, 240, ParserIcons.EffectAbyssalBlot);
+                }
+            }
+
+            // Abyssal Raze
+            if (log.CombatData.TryGetEffectEventsBySrcWithGUID(player.AgentItem, EffectGUIDs.RevenantSpearAbyssalRaze, out IReadOnlyList<EffectEvent> abyssalRazes))
+            {
+                var skill = new SkillModeDescriptor(player, Spec.Revenant, AbyssalRaze, SkillModeCategory.ShowOnSelect);
+                foreach (EffectEvent effect in abyssalRazes)
+                {
+                    uint radius = 180;
+                    long warningDuration = 800; // Overriding logged effect duration of 5100
+                    var connector = new PositionConnector(effect.Position);
+                    (long start, long end) lifespanWarning = (effect.Time, effect.Time + warningDuration);
+                    var circle = (CircleDecoration)new CircleDecoration(radius, lifespanWarning, color, 0.5, connector).UsingSkillMode(skill);
```

Comment:
Only the growing decoration is with filled false now, before both were filled false. Is the change intentional? If not adding UsingFilled(false) to circle and using AddDecorationWithGrowing instead should yield the same result as previous.

Reply:
Updated it again, hadn't noticed it with the debug on